jobs/payment-jobs/tasks/stale_payment_task.py

```diff
@@ -105,9 +105,9 @@ def _verify_created_direct_pay_invoices(cls):
         for invoice in created_invoices:
             try:
                 current_app.logger.info(f'Verify Invoice Job found records.Invoice Id: {invoice.id}')
-                paybc_invoice = DirectPayService.query_order_status(invoice, InvoiceReferenceStatus.ACTIVE.value)
+                paybc_invoice: OrderStatus = DirectPayService.query_order_status(invoice, InvoiceReferenceStatus.ACTIVE.value)
```

Comment:
don't do type hints like this, just set the type of the return of `query_order_status`, the IDE will infer it